Packs/Darktrace/IncidentFields/DarktraceIncidentFields.json

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
Incident Fields files should contain the a dict with the content of the incidentfield.
Please split you list of IncidentFields, that way that each incident field will have its own file and content.
for example:
for the incident id : incident_darktracecommentcount    the name of its file should be incidentfield-darktracecommentcount
and the content of the file should be:
{
			"id": "incident_darktracecommentcount",
			"version": 1,
			"modified": "2020-09-01T20:27:31.574123578Z",
			"sortValues": null,
			"packID": "",
			"itemVersion": "",
			"fromServerVersion": "",
			"toServerVersion": "",
			"vcShouldIgnore": false,
			"vcShouldKeepItemLegacyProdMachine": false,
			"commitMessage": "",
			"shouldCommit": false,
			"name": "Darktrace Comment Count",
			"prevName": "Darktrace Comment Count",
			"ownerOnly": false,
			"placeholder": "",
			"description": "Number of comments on the Darktrace model breach",
			"cliName": "darktracecommentcount",
			"type": "number",
			"closeForm": false,
			"editForm": true,
			"required": false,
			"script": "",
			"fieldCalcScript": "",
			"neverSetAsRequired": false,
			"isReadOnly": false,
			"selectValues": null,
			"validationRegex": "",
			"useAsKpi": false,
			"locked": false,
			"system": false,
			"content": false,
			"group": 0,
			"mergeStrategy": "",
			"hidden": false,
			"associatedTypes": [
				"Darktrace Model Breach"
			],
			"systemAssociatedTypes": null,
			"associatedToAll": false,
			"unmapped": false,
			"unsearchable": false,
			"caseInsensitive": true,
			"columns": null,
			"defaultRows": null,
			"sla": 0,
			"threshold": 72,
			"breachScript": "",
			"validatedError": ""
		}
You can also use demisto custom content in order to do so. If you incidentfields are saved in you demisto, you can go to settings->about->troubleshooting-> download custom content.
downloading it will automatically split each incidentfield to each own file as needed.

Reply:
Or use `demisto-sdk download` which will do it automatically for you.

Reply:
Adjusted! Let me know if this looks correct

Reply:
looks good, however the fromversion in all of the incident-fields is set currently to 5.0.0 , but the pack fromversion from `metadata.json` is 6.0
Please update the fromversion key in each incident fields to `6.0.0` - you can do that using `demisto-sdk format` command in this way: `demsito-sdk format -i <__your pack path__> -fv 6.0.0`

Reply:
Believe this should now be resolved!